_includes/info-slider.scss

```diff
@@ -42,8 +42,8 @@
         }
 
         &:focus {
-            outline: 0;
-            background: $color-focus-background;
+            outline-color: #2A71C3;
```

Comment:
$color-focus is beschikbaar…

Reply:
Dit is niet $color-focus, die heeft onvoldoende contrast om hier te werken. 

Ik denk dat dit op dit moment een uitzondering op de regel is, is het een idee om dit zo te laten?

Alternatieven: nieuwe variabele maken (`$color-focus-darker`?) of `$color-focus` updaten.

Reply:
Laat het voor nu ff zo

Reply:
Sowieso op enige moment rechttrekken maar voor nu goed.